feature-group-definitions/aria-attribute-reflection.dist.yml

```diff
@@ -0,0 +1,96 @@
+# Generated from: aria-attribute-reflection.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: ARIA attribute reflection (initial support)
+description: WAI-ARIA attributes have corresponding properties on `Element` and `ElementInternals` objects.
+spec: https://w3c.github.io/aria/#accessibilityroleandproperties-correspondence
+status:
+  baseline: low
+  baseline_low_date: 2023-10-24
+  support:
+    chrome: "103"
+    chrome_android: "103"
+    edge: "103"
+    firefox: "119"
```

Comment:
Matches `role` in https://developer.mozilla.org/en-US/docs/Web/API/Element#browser_compatibility 👍